akka-stream-tests/src/test/scala/akka/stream/scaladsl/RestartSpec.scala

```diff
@@ -462,12 +462,14 @@ class RestartSpec extends StreamSpec(Map("akka.test.single-expect-default" -> "1
       (minBackoff, maxBackoff, randomFactor, maxRestarts) ⇒ RestartFlow.withBackoff(minBackoff, maxBackoff, randomFactor, maxRestarts)
     }
 
-    def setupFlow(minBackoff: FiniteDuration, maxBackoff: FiniteDuration, maxRestarts: Int = -1, onlyOnFailures: Boolean = false) = {
+    def setupFlow(minBackoff: FiniteDuration, maxBackoff: FiniteDuration, maxRestarts: Int = -1, onlyOnFailures: Boolean = false): (AtomicInteger, TestPublisher.Probe[String], TestSubscriber.Probe[String], TestPublisher.Probe[String], TestSubscriber.Probe[String]) = {
```

Comment:
`TupleN+1`

Reply:
I take no credit for this awesome tuple

Reply:
ok but break the line at least please ;-)